demos/ragas-evaluation/rag-system-demo/README.md

```
@@ -0,0 +1,193 @@
+---
+
+## Table of Contents
+
+- [Initial Setup](#initial-setup)
+  - [Instructions to get GPUs on OpenShift](#instructions-to-get-gpus-on-openshift)
+  - [Update RHOAI DSC](#update-rhoai-dsc)
+  - [Create a Workbench](#create-a-workbench)
+- [Deploy AI Models and Llama Stack Server](#deploy-ai-models-and-llama-stack-server)
+- [Run the Evaluation Flow](#run-the-evaluation-flow)
+  - [Generate Synthetic Dataset](#generate-synthetic-dataset)
+  - [Run RAGAS Evaluation](#run-ragas-evaluation)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)